### DISCUSSION TITLE
Java runtime metrics?

### DISCUSSION THREAD

**Discussion by neplooho** [open-telemetry/opentelemetry-java-instrumentation#2247] [Q&A] [answered]
What is current state on metrics provided by opentelemetry javaagent? 
https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-metrics/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics
How can I enable them? I can't find any example on how to export metrics collected by javaagent

**Comment by anuraaga** [comment, marked as answer]
Hi @neplooho - we don't currently support runtime metrics in the javaagent. I've gone ahead and sent #2311 to enable them so they'd be there in the next release.

**Reply by ndeepakp** [reply]
@anuraaga  Is this enabled now? And can we have an example to use this?

**Reply by trask** [reply]
x-posted to https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/6530
